### ISSUE TITLE
Doesn't leave the meetings

### ISSUE THREAD

**Issue by MickeyArora11** [atharva-lipare/MS-Teams-Auto-Joiner#2]
Hi! I modified the values as you have instructed in the config.Json and still the meeting doesn't end even though the number of minimum participants are crossed i.e., 50 (Input value). If the meeting isn't left it won't join any other meeting, I've checked the code and even tried tweaking it by changing default minimum value to 50. Nothing works, I've checked the code twice before executing, the data is working as login happens smoothly, the meetings joining is accurate too. Only issue is that it doesn't leave when the organizer (admin person) hasn't left but the meeting has less than minimum participants. I have noticed that its because it doesn't open the active participants to scan it in certain intervals. I hope you understood my situation and can help out. Thank you!

**Comment by atharva-lipare** [comment]
> I have noticed that its because it doesn't open the active participants to scan it in certain intervals.

You're right. Have changed the click() method and tested it many times. Must have solved the Issue. Let me know whether you're still facing the issue at your end and then i will close the issue.

**Comment by HariAcidReign** [comment]
I think this issue is still existing. Joining automatically and Twilio work fine but it's not leaving the meeting when either the host leaves or when it falls below the threshold (I have it in 10) which kinda destroys the purpose of this project itself. Could you mind checking as to whether the auto-leaver is working? @atharva-lipare

**Comment by ghost** [comment]
The issue still exists

**Comment by prashantcoding** [comment]
what happen when somebody kicks you out

**Comment by atharva-lipare** [comment]
okay, will try finding the problem and for @prashantcoding, it would try joining again.
